fili-wikipedia-example/README.md

```diff
@@ -9,11 +9,19 @@ This example is an entirely self contained example that provides a Fili applicat
 1. Follow the [Druid instructions](http://druid.io/docs/0.9.1.1/tutorials/quickstart.html) to bring up a local Druid 
    cluster on your Unix based machine.
 2. Clone this repository to your computer.
+```bash
+git clone git@github.com:yahoo/fili.git
+```
```

Comment:
This needs to be indented under the list in order for the next list item to be 3 instead of 1 (ie. for the list to continue)